docs/reference/index.md

```diff
@@ -19,7 +19,7 @@ Logstash accelerates your insights by harnessing a greater volume and variety of
 You’ll use the {{ls}} [{{es}} output plugin](logstash-docs-md://lsr/plugins-outputs-elasticsearch.md) to send data to {{serverless-full}}.
 Note these differences between {{es-serverless}} and both {{ess}} and self-managed {{es}}:
 
-* Use **API keys** to access {{serverless-full}} from {{ls}}. Any user-based security settings in your [{{es}} output plugin](logstash-docs-md://lsr/plugins-outputs-elasticsearch.md) configuration are ignored and may cause errors.
+* Use [**API keys**](/reference/secure-connection.md#ls-api-keys) to access {{serverless-full}} from {{ls}}. Any user-based security settings in your [{{es}} output plugin](logstash-docs-md://lsr/plugins-outputs-elasticsearch.md) configuration are ignored and may cause errors.
```

Comment:
We can also be more direct and say that Serverless does not support native user authentication.  You must use API keys to access Serverless.